.env.example

```diff
@@ -1,5 +1,15 @@
+ENVIRONMENT=local
 API_CLIENT_ID=approved-premises-ui
 API_CLIENT_SECRET=clientsecret
-COMMUNITY_ACCOMMODATION_API_SERVICE_NAME=approved-premises
+SERVICE_SIGNIFIER=path
+APPROVED_PREMISES_SUBDOMAIN=approved-premises
+TEMPORARY_ACCOMMODATION_SUBDOMAIN=temporary-accommodation
 APPROVED_PREMISES_ROOT_PATH=approved-premises
 TEMPORARY_ACCOMMODATION_ROOT_PATH=temporary-accommodation
+COMMUNITY_ACCOMMODATION_API_SERVICE_NAME=temporary-accommodation
+WIREMOCK_ENDPOINT=http://localhost:9092
+
+# -- Wiremock on
```

Comment:
I still find these comments useful, do you? Happy to drop if not.

Reply:
Hmm, I'm not even sure WIREMOCK_ENDPOINT does anything? I can't find any reference to it in code. And I generally switch between running via `script/server` (wiremock) and `ap-tools` (full stack) without needing to touch this. Are we sure these still do anything?

Reply:
You'll know better than I 👍 I'll drop this.